unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,64 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const result1 = evenNumbers([1, 4, 5, 10, 0]);
+    expect(result1).toEqual([4, 10, 0]);
+    const result2 = evenNumbers([]);
```

Comment:
It's best to write 3 separate tests than combine them all into one test. It makes debugging a lot easier since then you know which tests succeed and which don't.

The way most unit test frameworks work, if the first `expect` fails, an exception is thrown and the code stops running. So you lose information when you put one test after the other.